pmoves/Makefile

```
@@ -4179,6 +4179,74 @@ down-external: ## Stop external integration services
+# Services fall back to their own password prompts (Open Notebook's
+# RemoteUserAuthMiddleware is fail-closed and needs the Traefik-injected
+# X-Forward-Auth-Secret header, so a direct hit gets the password check by design).
+#
+# Both overlays go up together on purpose: the auth.pmoves.ai router is declared on
+# the traefik container but points at `sso-auth@docker`, so Traefik without sso-auth
+# is an edge whose own login route 404s.
+# ---------------------------------------------------------------------------
+EDGE_DC := docker compose -p $(PROJECT)-edge --project-directory $(CURDIR) $(COMPOSE_ENV_FILES) -f docker-compose.traefik.yml -f docker-compose.sso.yml
+
+.PHONY: up-edge
+up-edge: ensure-env-shared ## Start the Traefik edge + sso-auth (media./auth.pmoves.ai, forward-auth)
```

checkmake (0.3.2)

[warning] Target body for "up-edge" exceeds allowed length of 5 lines (13).

(maxbodylength)